cockatrice/src/gametimefilter.h

```diff
@@ -1,65 +0,0 @@
-#ifndef COCKATRICE_GAMETIMEFILTER_H
-#define COCKATRICE_GAMETIMEFILTER_H
-
-#include <QDebug>
-#include <QVector>
-
-// If this enum is ever expanded, make sure to update:
-// - getMaxGameAgeEnumOptions (order shown is order presented to user)
-// - text option generation in getMaxGameAgeInSecondsFromEnum
-// - validation conditions in getMaxGameAgeInSecondsFromComboBoxIndex
-enum MaxGameAge
-{
-    NO_MAX_AGE = 0,
```

Comment:
I don't like enums :upside_down_face:

Reply:
Really? Enums are great imo

Reply:
not for making changes later, consider you want to add 20 minutes in between while not breaking everyone's current settings